doc/source/raysgd/v2/user_guide.rst

```diff
@@ -40,7 +40,7 @@ RaySGD abstracts away the complexity of setting up a distributed training system
             results = model(x)
         return results
 
-To convert this to RaySGD, we add a `config` parameter to `train_func()`:
+To convert this to RaySGD, we add a ``config`` parameter to ``train_func()``:
```

Comment:
`config` parameter is optional

Reply:
I moved this out of the `Quick Start` and into ` Configurations` section. Will fix the Quick Start to actually run in a following PR.